backend/src/controllers/auth/logout.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from 'express';
+
+import { TMiddlewareCall } from 'interfaces/commonMiddleware';
+import { isError } from 'utils/typeGuards/isError';
+import { verifyAccessToken } from 'utils/auth/authUtils';
+import { clearTokenProvider } from 'db/providers/authProvider';
+
+const logout = async (req: Request, res: Response, next: TMiddlewareCall) => {
+  try {
+    const accessToken = req.headers.authorization?.split(' ')[1];
```

Comment:
I haven't actually gone through all the project code yet, but in my opinion it would be great to implement an util function, that extracts token from auth header string. It's not really hard to implement but it would make code much clearer and we would solve the problem of code duplication. 🤔